### ISSUE TITLE
Firefox Menu, Addressbar, Tab tree is all black & not visible

### ISSUE THREAD

**Issue by sagnikpal** [traxium/tabtree#85]
I have been using Tab tree for quite some time now. 
Today suddenly I see that Tab Tree (I keep it at left side of window) the menu bar & address bar are showing black & not visible.
Once I disable tabtree the menu, addressbars & tabs become visible.
Refer to attached screenshots.
![tabtree disabled](https://cloud.githubusercontent.com/assets/16877106/12973053/0731d2bc-d0d0-11e5-84f8-e16865839945.png)
![tabtree enabled](https://cloud.githubusercontent.com/assets/16877106/12973054/0734e0ce-d0d0-11e5-8b92-8438fe1bcebb.png)

So far I have tried following:
1) Uninstalled & Re-installed firefox - Did not solve issue
2) Unistalled & re-installed tab tree latest version - Did not solve issue
3) Unistalled & re-installed tab tree previous to latest version - Did not solve issue
4) The famous windows restart ;-) - Did not solve issue

Anyone else facing the issue?

**Comment by traxium** [comment]
Yes #12 #53 #72. Do you have Intel graphics card?

> try disabling "Use hardware acceleration when available" in Firefox options (Options -> Advanced -> General)

**Comment by traxium** [comment]
I just made the warning message about this on [AMO](https://addons.mozilla.org/en-US/firefox/addon/tab-tree/) more noticeable.
By the way, did you install any new software (or did something else) before this happened?

**Comment by sagnikpal** [comment]
Hi, 

Disabling "Use hardware acceleration when available" in Firefox options (Options -> Advanced -> General) solved the issue. Thanks.

To answer your other question<--did you install any new software (or did something else) --> 
Yes, I had updated my graphics driver recently. 
Windows update had pushed a generic graphics driver which had messed my screen resolution. So I downloaded the graphics driver from hp.com & updated. 

Guess this is another learning for me. Never knew graphics driver update could cause problem with only a section of an application & not throw whole screen.
